parameter-alias.js

```diff
@@ -1,21 +0,0 @@
-import DS from 'ember-data';
```

Comment:
glad to see this go tbh

Reply:
this was never doing anything lol its in the root dir

Reply:
alias still needs to exist - it's in `app/models`

Reply:
thaaaaat makes more sense